Applications/MainApplication/Gui/MaterialEditor.cpp

```diff
@@ -96,17 +99,14 @@ void MaterialEditor::newKsColor( const QColor& color ) {
     if ( m_renderObject && m_usable )
     {
         Core::Utils::Color c( color.redF(), color.greenF(), color.blueF(), 1.0 );
-        m_material->m_ks = c;
+        m_blinnphongmaterial->m_ks = c;
```

Comment:
in all these simple affectation, why not allowing the compiler to construct the object in place by doing `m_blinnphongmaterial->m_ks = { color.redF(), color.greenF(), color.blueF(), 1.0 };`

Reply:
Thanks. Updated.